RTTR/texte/de_DE/readme.txt

```diff
@@ -1,344 +1,360 @@
-         Ein Punktesystem ist vorgesehen, aber noch nicht implementiert.
+    a) Direktes Spielen
+       Die Erstellung eines Multiplayerspiels erfolgt genauso wie bei 1. Die
+       weiteren Spieler gehen bei "Direkte IP" unter "Spiel beitreten" und
+       geben die IP oder den Hostnamen des Spielerstellers (Host) ein.
 
-         Alle eingegebenen Daten werden nicht weitergegeben und dienen rein
-         zur Identifikation des Spielers. Die zur Registrierung notwendige
-         Emailadresse ist momentan nur ggf. zum Zusenden eines neuen
-         Passworts gedacht und ist nicht öffentlich einsehbar.
+    b) Spielen über integrierte Lobby
+       Über die eingebaute Lobby kann man vorhandene Spiele einsehen,
+       diese eröffnen und auch beitreten. Weiterhin kann man sich hier
+       mit anderen Spielern direkt unterhalten.
 
-      Für das Spiel wird Port 3665 (TCP) genutzt. Dieser muss, falls man selbst
-      das Spiel eröffnen möchte, u.a. bei Verwendung eines Routers, einer
-      Firewall usw. freigegeben werden!
-      Router nennen diese Einstellung "Virtual Server" oder einfach nur
-      "Port Forwarding". Dort ist dann Port 3665 vom Typ TCP auf die interne IP
-      des Spiel-PCs weiterzuleiten.
+       Es wird ein Foren-Account von http://www.siedler25.org benötigt, dessen
+       Login-Daten für den Login in die Spiel-Lobby verwendet werden.
 
-      Die Schneckensymbole im oberen Teil stehen für Lags der jeweiligen
-      Spieler.
+    c) LAN Spiele
+       Der LAN-Bereich ist ähnlich der Internet-Lobby funktioniert aber über das
+       lokale Netz und benötigt kein Login. Es können auch Programme wie Hamachi
+       verwendet werden um virtuelle LANs über das Internet zu erstellen.
 
-      Wenn der Host das Spiel verlässt, können die übrigen Spieler noch nicht
-      das Spiel weiter fortführen.
+    Für das Spiel wird Port 3665 (TCP) genutzt. Dieser muss, falls man selbst
+    das Spiel eröffnen möchte, u.a. bei Verwendung eines Routers, einer
+    Firewall usw. freigegeben werden!
+    Router nennen diese Einstellung "Virtual Server" oder einfach nur
+    "Port Forwarding". Dort ist dann Port 3665 vom Typ TCP auf die interne IP
+    des Spiel-PCs weiterzuleiten. Bei der Verwendung einer lokalen Firewall
+    muss auch hier der Zugriff zugelassen werden.
+    Im LAN Modus werden Ports 3666 und 3667 (UDP) zum Finden von Spielen verwendet.
+    
+    Es ist nicht notwendig, dass jeder Spieler die Karte hat. Diese wird bei
+    Spielbegin automatisch vom Host übertragen.
 
-      Es sind sowohl Spiele über das Internet als auch über LAN möglich, wenn
-      die jeweilige IP bekannt ist.
+    Die Schneckensymbole im oberen Teil stehen für Lags der jeweiligen
+    Spieler.
 
-      Das Übertragen der Karte geschieht immer automatisch (auch wenn die
-      Spielteilnehmer die Karte des Hosts schon besitzen).
+    Wenn der Host das Spiel verlässt, können die übrigen Spieler noch nicht
+    das Spiel weiter fortführen.

```

Comment:
Das ist etwas missverständlich. Sollte da in etwa stehen, dass die übrigen Spieler das Spiel nicht weiter fortführen können, wenn der Host das Spiel verlässt?